UKHomeOffice/nmsw-ui
# NMSW-989 FAL5 and FAL 6 changes required

# Ticket

NMSW-989

----

## AC

Rename FAL5
- FAL5 currently named - "Crew details including supernumeraries FAL 5"
- need to be changed to - "Crew Details (FAL 5)" (remove supernumeraries)

Rename FAL6  
- FAL6 currently named - "Passenger details"
- need to be changed to "Passenger details including supernumeraries (FAL 6)"

----

## To test

- Run app
- On landing page
- > Templates should have updated names
- > Templates should download correctly
- Sign in
- Go to /templates
- > Templates should have updated names
- Click report a voyage
- Upload FAL 1 and continue
- On tasklist page
- > Tasklist should have updated names
- Click on Crew details
- > Title should have correct name
- Go back to tasklist
- Click on passenger details
- Click yes to passengers
- > Title should have correct name
- Go back and select no to passengers
- Click check your answers
- > The uploads section should have updated names for passenger part
- Click on the help tab
- > Templates should have updated names

----

## Notes
